src/main/resources/hudson/plugins/jira/JiraSite/config.jelly

```diff
@@ -1,5 +1,5 @@
 <?jelly escape-by-default='true'?>
```

Comment:
Existing username/password will migrated to credentials and be stored. The config will not be touched anyway, i.e. require end user re-config jira-plugin global settings. It will be better that no re-config required. I'm struggling on this yet. I'm not sure how to trigger jenkins to auto save jira-plugin settings.

Reply:
@warden @olamy @oleg-nenashev any ideas on this?